indexd_ffi/Cargo.toml

```diff
@@ -23,7 +23,7 @@ sia_sdk = { version = "0.1.1", path = "../sia_sdk" }
 thiserror = "2.0.17"
 tokio = { version = "1.47.1", features = ["rt-multi-thread", "sync"] }
 tokio-util = "0.7.16"
-uniffi = { version = "0.30.0", features = ["cli", "tokio"] }
+uniffi = { version = "0.29.4", features = ["cli", "tokio"] }
```

Comment:
Did it cause issues?

Reply:
Yeah. We need to keep it pinned to this https://github.com/jhugman/uniffi-bindgen-react-native/blob/main/Cargo.toml or we get weird hard to debug errors.